src/GridRegistry.ts

```diff
@@ -0,0 +1,47 @@
+import { WidgetBaseConstructor } from '@dojo/widget-core/interfaces';
+import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
+import Body, { BodyInterface } from './Body';
+import Cell, { CellInterface } from './Cell';
+import Footer, { FooterInterface } from './Footer';
+import Header, { HeaderInterface } from './Header';
+import HeaderCell, { HeaderCellInterface } from './HeaderCell';
+import Row, { RowInterface } from './Row';
+
+export interface GridRegistered {
+	[key: string]: WidgetBaseConstructor;
+	body: BodyInterface;
```

Comment:
I was trying to avoid forcing users to subclass Body itself. As long as it confirms to properties/children, I want any subclass of WidgetBase to work.